src/pkg/services/m365/api/users.go

```diff
@@ -35,6 +36,50 @@ type Users struct {
 
 type UserInfo struct {
 	DiscoveredServices map[path.ServiceType]struct{}
+	HasMailBox         bool
+	HasOneDrive        bool
+	Mailbox            MailboxInfo
+}
+
+type MailboxInfo struct {
+	Purpose                    string
+	ArchiveFolder              string
+	DateFormat                 string
+	TimeFormat                 string
+	DelegateMeetMsgDeliveryOpt string
+	Timezone                   string
+	AutomaticRepliesSetting    AutomaticRepliesSettings
+	Language                   Language
+	WorkingHours               WorkingHours
+	ErrGetMailBoxSetting       clues.Err
```

Comment:
Can this get a TODO to be removed in favor of standard error handling?

Also, this should be an `error`, not a `clues.Err`.  We're not persisting the value (you'd want a clues.ErrCore for that, anyway), therefore no concrete struct is required.  If we need to extract the clues data later on, we can still do that with a generic `error`.
```suggestion
	ErrGetMailBoxSetting       error
```